test/e2e/localnet-underlay.go

```diff
@@ -52,7 +52,7 @@ func teardownUnderlay(ovsPods []v1.Pod) error {
 	return nil
 }
 
-func ovsPods(clientSet clientset.Interface) []v1.Pod {
+func ovsPods(clientSet clientset.Interface) (error, []v1.Pod) {
```

Comment:
oh, no. 

Please re-use the common pattern of having the error as the second argument.
Everything in gomega is built to use it - for instance, if the signature is `func ovsPods(clientSet clientset.Interface) ([]v1.Pod, error)`  and the function fails when invoking, it will be caught by `Expect(ovsPods(...)).NotTo(BeEmpty())`.

Reply:
ha, absolutely. my bad in not paying attention....